lib/dry/logic/predicates.rb

```diff
@@ -3,6 +3,7 @@
 require "bigdecimal"
 require "bigdecimal/util"
 require "date"
+require "open-uri"
```

Comment:
Shouldn't this just be `require "uri"`? I don't think we need to bring in the opening behavior just to get to a regexp within the URI module. 

Would you mind giving that change a try? Thanks!

Reply:
Works as well.